pkg/controller/daemonset/daemonset_controller.go

```diff
@@ -755,7 +755,8 @@ func (dsc *ReconcileDaemonSet) syncWithPreparingDelete(ds *appsv1alpha1.DaemonSe
 			podsCanDelete = append(podsCanDelete, podName)
 			continue
 		}
-		if updated, gotPod, err := dsc.lifecycleControl.UpdatePodLifecycle(pod, appspub.LifecycleStatePreparingDelete); err != nil {
+		requiredPodNotReady := ds.Spec.Lifecycle.PreDelete.MarkPodNotReady
```

Comment:
can we rename all requiredPodNotReady variables/comments to markPodNotReady

Reply:
done